app/controllers/profiles_controller.rb

```diff
@@ -17,7 +17,18 @@ def show
   def tooltip
     expires_in 1.minute
 
-    render layout: false
+    respond_to do |format|
+      format.json do
+        render json: {
+          html: render_to_string(
+            partial: "tooltip",
+            locals: { user: @user, profile: @profile },
+            layout: false,
+            formats: [:html]
+          )
+        }
+      end
+    end
```

Comment:
What's the benefit of doing it this way over the original version?

Reply:
I have the same question. I don't mind it if there's a good enough reason, but it does feel like extra noise on the server side unless we need to do this. I'll wait to understand this before merging.

Reply:
@ErikSchierboom Just chatting to Karlo about this. Basically we ReactQuery expects everything as JSON and adding the exceptions to that causes problems. I'm going to add a helper though so that we don't have to write all that code out every time we want to render a template as JSON :)

I'll add that to this PR then merge it.

Reply:
Okay, sounds good!